markdown/ch6.markdown

```diff
@@ -359,41 +372,105 @@ x = Dinv2 @ eigvecs[:, 1]
 ```
 
 (The reasons for using this vector are too long to explain here, but appear in
-the paper's supplementary material, linked above.)
+the paper's supplementary material, linked above. The short version is that
+choosing this vector minimizes the total length of the links between neurons.)
+
+There is one small kink that we must address before proceeding: eigenvectors
+are only defined up to a multiplicative constant. This follows simply from the
+definition of an eigenvector: suppose $v$ is an eigenvector of the matrix $M$,
+with corresponding eigenvalue $\lambda$. Then $\alpha v$ is also an eigenvector
+of $M$ for any scalar number $\alpha$, because
+$M(\alpha v) = \alpha Mv = \alpha \lambda v = \lambda (\alpha v)$. So, it is
```

Comment:
How about a minor tweak:

```
because if $Mv = \lambda v$ then it is also true that $M (\alpha v) = \lambda (\alpha v)$
```

This way you can immediately see the similarity in form to the previous definition given for eigenvectors.